pdcupdater/handlers/modules.py

```diff
@@ -185,13 +185,13 @@ def get_or_create_unreleased_variant(self, pdc, body):
         variant_release =  body['version'] # This is supposed to be equal to release
         variant_uid = "%s-%s-%s" % (variant_id, variant_version, variant_release)
 
-        try:
-            unreleased_variant = pdc['unreleasedvariants'][variant_uid]._()
-        except beanbag.BeanBagException as e:
```

Comment:
The new code does not raise in case when module cannot be found, it just returns empty list.

Reply:
Cool - can you add a `log.warn` when it is not found, so we can track it if it misbehaves again?

Reply:
You would find that in `if not unreleased_variants:` block. When it is not found, it is going to be created, so the log.warn wouldn't make sense for this case. And when there is another problem with communication with PDC, the pdcclient raises an exception as it used to do.

Reply:
And in case it is not found by a bug in PDC/PDCClient, the self._create_unreleased_variant code will fail the same way as it fails currently :).

Reply:
:+1: cool.